webapp/src/components/BasicInput/BasicInput.stories.js

```diff
@@ -0,0 +1,17 @@
+import BasicInput from './BasicInput.vue';
+
+export default {
+    title: 'Example/BasicInput',
+    component: BasicInput,
```

Comment:
Adding this here:
```js
  argTypes: {
    'createItem': { action: 'create-item' }
  }
```

And then render your `BasicInput` like this:

```
const Template = (args, {argTypes}) => ({
    props: Object.keys(argTypes),
    components: {BasicInput},
    template:
        '<BasicInput @create-item="createItem"/>',
});
```

Will configure `Actions` tab:

![Screenshot - 2020-11-11T211058 620](https://user-images.githubusercontent.com/2110676/98859410-6ed85b80-2462-11eb-8c82-eeb3fa7fe1ed.png)